src/main/java/io/kestra/plugin/ollama/cli/OllamaCLI.java

```diff
@@ -0,0 +1,156 @@
+package io.kestra.plugin.ollama.cli;
+
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.*;
+import io.kestra.core.models.tasks.runners.TaskRunner;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.scripts.exec.scripts.models.DockerOptions;
+import io.kestra.plugin.scripts.exec.scripts.models.ScriptOutput;
+import io.kestra.plugin.scripts.exec.scripts.runners.CommandsWrapper;
+import io.kestra.plugin.scripts.runner.docker.Docker;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.Valid;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import jakarta.validation.constraints.NotNull;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Execute Ollama commands to interact with LLM models."
+)
+@Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Pull a model and generate text completion",
+            code = """
+                id: ollama_flow
+                namespace: company.team
+
+                tasks:
+                  - id: ollama_cli
+                    type: io.kestra.plugin.ollama.cli.OllamaCLI
+                    commands:
+                      - ollama pull llama2
+                      - ollama run llama2 "Tell me a joke about AI" > completion.txt
+                    outputFiles:
+                      - completion.txt
+                """
+        ),
+        @Example(
+            full = true,
+            title = "List available models and output as JSON",
+            code = """
+                id: ollama_list_models
+                namespace: company.team
+
+                tasks:
+                  - id: list_models
+                    type: io.kestra.plugin.ollama.cli.OllamaCLI
+                    commands:
+                      - ollama list --format json > models.json
+                    outputFiles:
+                      - models.json
+                """
+        )
+    }
+)
+public class OllamaCLI extends Task implements RunnableTask<ScriptOutput>, NamespaceFilesInterface, InputFilesInterface, OutputFilesInterface {
+    private static final String DEFAULT_IMAGE = "ollama/ollama";
+
+    @Schema(
+        title = "The commands to run."
+    )
+    @NotNull
+    protected Property<List<String>> commands;
+
+    @Schema(
+        title = "Additional environment variables for the current process."
+    )
+    protected Property<Map<String, String>> env;
+
+    @Schema(
+        title = "Deprecated, use 'taskRunner' instead"
+    )
+    @PluginProperty
+    @Deprecated
+    private DockerOptions docker;
```

Comment:
It doesn't seem used, plus it's deprecated and you already used `taskRunner`, so I think you could remove it.